scipy/stats/tests/test_stats.py

```diff
@@ -3243,6 +3244,19 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    @dec.knownfailureif(True, "Tough NIST Test Cases fail")
```

Comment:
I think it would be better to only add knownfailure the the tough cases that fail, so the other cases raise a test failure if they fail.